features/tags_and_wrangling/tag_wrangling_more.feature

```diff
@@ -141,10 +141,83 @@ Feature: Tag wrangling: assigning wranglers, using the filters on the Wranglers
        And I press "Wrangle"
-     Then I should see "The following tags were successfully wrangled to Cowboy Bebop: Faye Valentine is a sweetie, Ed is a sweetie Wrangle Tags for Cowboy Bebop"
+     Then I should see "The following tags were successfully wrangled to Cowboy Bebop: Faye Valentine is a sweetie, Ed is a sweetie"
        And the "Faye Valentine is a sweetie" tag should be canonical
        And the "Ed is a sweetie" tag should be canonical
 
+  Scenario: more wrangling....Freeform
+    Given the following typed tags exists
+        | name                                   | type         | canonical |
+        | Cowboy Bebop                           | Fandom       | true      |
+        | Faye Valentine is a sweetie            | Freeform     | false     |
+        | Ed is a sweetie                        | Freeform     | false     |
+        | Spike Spiegel is a sweetie             | Freeform     | false     |
+      And a media exists with name: "Anime & Manga", canonical: true
+      And I am logged in as a random user
+      And I post the work "Brain Scratch" with fandom "Cowboy Bebop" with freeform "Spike Spiegel is a sweetie"
+      And I post the work "Asteroid Blues" with fandom "Cowboy Bebop" with freeform "Faye Valentine is a sweetie"
+     When the tag wrangler "lain" with password "lainnial" is wrangler of "Cowboy Bebop"
+       And I go to the tagwranglings page
+       And I follow "Fandoms by media (2)"
+       And I check the wrangling option for "Cowboy Bebop"
+       And I press "Wrangle"
+     Then I should not see "Cowboy Bebop"
+
+  Scenario: more wrangling....Relationships
+    Given the following typed tags exists
+        | name                                   | type         | canonical |
+        | Tybalt (Toby Daye)                     | Character    | false     |
+        | October Daye                           | Character    | false     |
+        | Toby Daye/Tybalt                       | Relationship | true      |
+        | October Daye Series - Seanan McGuire   | Fandom       | false     |
+      And a media exists with name: "Books & Literature", canonical: true
+      And I am logged in as a random user
+      And I post the work "The course of true love never did run smooth" with fandom "October Daye Series - Seanan McGuire" with character "Tybalt (Toby Daye)" with second character "October Daye" with relationship "Toby Daye/Tybalt"
+     When the tag wrangler "lain" with password "lainnial" is wrangler of "October Daye Series - Seanan McGuire"
+       And I go to the tagwranglings page
+       And I follow "Relationships by fandom (1)"
+       And I check the wrangling option for "Toby Daye/Tybalt"
+       And I fill in "fandom_string" with "October Daye Series - Seanan McGuire"
+       And I press "Wrangle"
+     Then I should see "The following names are not canonical fandoms: October Daye Series - Seanan McGuire."
+
+  Scenario: more wrangling....Relationships canonically
```

Comment:
Same thing --  I think we need a better description for the scenario. What I see happening is we're making sure a relationship can be wrangled into a canonical fandom from the relationships bin, but I'm not sure

- Why the character tags are necessary
- Why the media tag is necessary
- Why the work is necessary